packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/components",
-  "version": "0.101.4",
+  "version": "0.101.5-fb-requires-permission.0",
```

Comment:
I would think this would be a 0.102.0 version since you're adding and removing components that are exported.

Reply:
I just do the minimal increment until I'm ready to release. Yes, this will be 102 if that's the increment.